### ISSUE TITLE
Charging Luna2000-7-s1 using Nordpool lowest Spot Prices

### ISSUE THREAD

**Issue by wachaudhary** [biemond/solaredge.modbus#114]
Hi, 

I would like to inquire about the possibility of adding a new feature to the app that would allow users to schedule the charging of their Huawei Luna2000-7-S1 battery based on the lowest electricity prices from Nord Pool. This integration would enable the app to automatically adjust the charging times according to real-time spot prices, ensuring that users can optimize their energy costs by charging their battery during the most economical periods.

I believe this feature would significantly enhance the app's functionality and provide great value to Homey Pro users, especially those managing energy storage solutions. By leveraging Nord Pool's spot pricing data, this capability could provide substantial cost savings and offer a more intelligent, automated energy management system for users.

**Comment by biemond** [comment]
yes awesome. but for legal and for any responsibility claims plus other apps are better in spot prices you have to do this yourself.  

I only want to provide you the flow actions to trigger something for charing etc  when the modbus docs allows me .  

but looks like new energy homey tab , it will go that way . I hope to integrate with that so you can use that.
